fuzz/target.c

```diff
@@ -446,6 +446,8 @@ fuzz_eager_output(const uint8_t *data, size_t size)
 
 	size_t max_pattern_length = 0;
 
+	const unsigned seed = size == 0 ? 0 : data[0];
```

Comment:
so I guess we'll srand() here

Reply:
Yes, I'll add that before I switch from a draft PR.

Reply:
This is now done in e133a74 on #29.